bin/androguard

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Androguard is a full Python tool to play with Android files."""
+
+# core modules
+import sys
+
+# 3rd party modules
+import click
+
+# local modules
+import androguard
+from androguard.cli import androaxml_main
+
+
+@click.group(help=__doc__)
+@click.version_option(version=androguard.__version__)
+def entry_point():
+    pass
+
+
+@entry_point.command()
+@click.option(
+    '--input', '-i', 'input_',
+    help='AndroidManifest.xml or APK to parse (legacy option)',
+)
+@click.option(
+    '--output', '-o',
+    help='filename to save the decoded AndroidManifest.xml to',
+)
+@click.argument(
+    'file_',
+    # help='AndroidManifest.xml or APK to parse',
+    required=False,
+)
+def androaxml(input_, output, file_):
+    """
+    Parse the AndroidManifest.xml.
+
+    Parsing is either direct or from a given APK and prints in XML format or
+    saves to file.
+
+    This tool can also be used to process any AXML encoded file, for example
+    from the layout directory.
+
+    Example:
+
+    \b
+        $ androguard androxml AndroidManifest.xml
+    """
+    if file_ is not None and input_ is not None:
+        print("Can not give --input and positional argument! "
+              "Please use only one of them!")
+        sys.exit(1)
+
+    if file_ is None and input_ is None:
+        print("Give one file to decode!")
+        sys.exit(1)
+
+    if file_ is not None:
+        androaxml_main(file_, output)
+    elif input_ is not None:
+        androaxml_main(input_, output)
+
+
+if __name__ == '__main__':
```

Comment:
afaik the click doc says to use entry_points in the setup.py instead. can you do this as well?

Reply:
Interesting catch! I was not aware of this. I want check how to use this in one of my projects first, then I'll adapt. I think I'll simply move `bin/androguard` to `androguard/entry_point.py`.

Reply:
I think having it in `bin/androguard` is fine. I think you can even have the __name__ stuff in there to be even more flexible - as long as `if __name__ ...` calls only a single function, which is also used as entry point.